components/FaceDetection/FaceDetection.js

```diff
@@ -0,0 +1,122 @@
+import { useRef, useEffect, useState } from "react";
+import * as Styled from "./FaceDetection.styled";
+import * as faceapi from "face-api.js";
+import { nanoid } from "nanoid";
+import SmileTrainer from "@/components/SmileTrainer/SmileTrainer";
+
+export default function FaceDetection() {
+  // const [emotions, setEmotions] = useState([]);
```

Comment:
```suggestion
```

Reply:
should stay in because FaceDetection component will be used for other purposes in the future and then we could need the state.